postgres/statements/permissions.go

```diff
@@ -0,0 +1,7 @@
+package statements
+
+const (
+	READ   string = "read"
+	WRITE  string = "write"
+	DELETE string = "delete"
+)
```

Comment:
All letters don’t need to be capitalized just the first letter

Reply:
removed permisions.go
is not used on adapters